src/System.Private.Reflection.Core/src/System.Private.Reflection.Core.Experimental.csproj

```diff
@@ -3,6 +3,7 @@
     <AssemblyName>System.Private.Reflection.Core.Experimental</AssemblyName>
     <EcmaMetadataSupport>true</EcmaMetadataSupport>
     <JitSupport>true</JitSupport>
+    <InterpreterSupport>true</InterpreterSupport>
```

Comment:
I'm wondering if we should just call this `ExperimentalSupport`/`DynamicCodeSupport`, or something like that, and replace `JitSupport` with that single flag. It doesn't seem like these assemblies need to know the details of the execution strategy, besides knowing that they have to reference the `.Experimental` flavors of the assemblies.

Reply:
Sounds good

Reply:
On second look, it seems there are some parts of `S.P.Typeloader` that are conditionally compiled based on the `SUPPORT_JIT` constant being set when `JitSupport` is `true`. We ideally should not have these parts when we're just interested in having an interpreter which will be the case if we use a single tag to encompass both JIT and Interpreter support

Reply:
On a third look, nevermind 😄